ivy/data_classes/container/experimental/losses.py

```diff
@@ -0,0 +1,332 @@
+# global
+from typing import Optional, Union, List, Dict
+
+# local
+import ivy
+from ivy.data_classes.container.base import ContainerBase
+
+
+class _ContainerWithLosses(ContainerBase):

```

Comment:
This should be renamed to indicated that it is an experimental instance
```suggestion
class _ContainerWithLossesExperimental(ContainerBase):

```

Reply:
Sure, @hirwa-nshuti : Thanks for the suggestion. I've resolved it